tgui/packages/tgui/components/RestrictedInput.js

```diff
@@ -18,7 +18,7 @@ const getClampedNumber = (value, minValue, maxValue) => {
   if (!value || !value.length) {
     return String(minimum);
   }
-  let parsedValue = parseInt(value.replace(/\D/g, ''), 10);
+  let parsedValue = parseInt(value.replace(/[^(\-|\d)]]/, ''), 10);
```

Comment:
```suggestion
  let parsedValue = parseInt(value.replace(/[^\-\d]/g, ''), 10);
```

Reply:
old one _did_ work but I tested in a different codebase